### DISCUSSION TITLE
Ground.type = none and Air.type = none QOL updates

### DISCUSSION THREAD

**Discussion by Amidweiz** [ikemen-engine/Ikemen-GO#965] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

Currently with `Ground.type = none` and `Air.type = none` in the hitdef are practically useless if one wants to make a hitdef attack that does not put the opponent into hit stun but is able to damage the opponent without the need for a redirection.

### Describe the solution you'd like

I think these new updates should be effected by the wip `ikemenversion` trigger if past mugen characters used it's original functionality in the past but the changes I would like to see are

- Being able to apply damage/chip damage to the opponent rather than nothing
- Being able to activate the `movecontact, movehit,` and `moveguarded` triggers. Currently it returns nothing
- Not needing to address `air.type` in the same hitdef since it does not take the `ground.type` value by default`

### Search terms

hitdef ground.type air.type

**Comment by potsmugen** [comment, marked as answer]
The second one sounds the most useful. Once you got those triggers active, you could do anything with RedirectID.
Also the whole type = none thing would become obsolete if you had another way of checking if the hitboxes are overlapping (and with which player's).

**Comment by K4thos** [comment]
Marked as answered since we do have clsn trigger now
